xml/System.Security.Cryptography/IncrementalHash.xml

```diff
@@ -54,8 +54,8 @@
         <ReturnType>System.Security.Cryptography.HashAlgorithmName</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets the name of the algorithm being performed.</summary>
+        <value>The name of the algorithm being performed.</value>
```

Comment:
I initially was correcting this to say "well, for HMAC it's the underlying hash".  But that isn't true.  Maybe we want to say that HMAC algorithms are prepended with "HMAC" to distinguish them from an unkeyed digest?  (If `HashAlgorithmName.SHA256` was passed to CreateHash, then `HashAlgorithmName.SHA256`. If `HashAlgorithmName.SHA256` was passed to CreateHMAC, then a HashAlgorithmName whose Name is "HMACSHA256")